sdn_tests/testplans/Stress_Latency_Test_Plan.md

```diff
@@ -0,0 +1,385 @@
+    </tr>
+    <tr>
+      <td><em>/components</em></td>
+    </tr>
+    <tr>
+      <td><em>/qos</em></td>
+    </tr>
+    <tr>
+      <td><em>/system</em></td>
+    </tr>
+    <tr>
+      <td><em>/lacp</em></td>
+    </tr>
+  </tbody>
+</table>
+
+The following test cases randomly select the set of paths from the above supported subtrees, the tests also randomly select the gNMI operations to validate the robustness of the system.\
+The key values are also randomly selected based on the testbed information. For example, the interface key for the interface module paths are randomly selected from the list of available interfaces in the testbed. The list of paths are randomly selected from the supported paths in go/gpins-openconfig.
+
+The infrastructure provides a fuzzing API to randomly select the list of paths, keys and gNMI operations to be validated along with the expected responses. The set of paths, keys, payload and expected response messages are provided as the artifact to the test cases.
+
+The list of paths will also include the paths which are not supported by the UMF and invalid keys. These paths are used in the test cases to validate the negative testing and gNMI server's performance metrics.
+
+The first few runs of these stress tests are used to establish the baseline benchmarks for each gNMI operation and for each test case. The baseline benchmarks are stored in the table along with the list of gNMI paths, payloads, keys and expected response. There will be another test which runs at the end of each test which analyzes the performance report against the baseline benchmarks for that system.The test will fail if the performance exceeds the benchmarks.
+
+We use up to 2 pictor clients, 6 SFE clients( 3 over inband and 3 over outerband ) for the gNMI service. The test plan assumes that max needed clients are 5(2 set and subscribe clients and 3 get/subscribe clients) for the stress testing based on the production use case.
+
+**Challenges:**
+
+-   The test infrastructure needs to support creating multiple gNMI clients and establish the connection to the gNMI server.
+-   The test infrastructure/gNMI server should provide an ability to turn off the master arbitration feature so that more than one client can simultaneously configure the OC paths for the purpose of stress testing. Using the same connection ID from both set clients will allow both clients to do the gNMI set requests.
+-   The paths, payload for the paths and expected response messages should be stored in the map along with supported gNMI operations so that we can use fuzzing to randomly select the path, gNMI operations for the below test cases.
+
+All the below tests will do the global set replace with a sanitized configuration prior to doing the actual test so that the testing payload will be guaranteed to be different from the current configurations. All the number such as 10 request or 5 clients are configurable for each test case
+
+### Infrastructure
```

Comment:
This section (### Infrastructure) can be removed. The paths are in openconfig at this point so no augment is necessary. It also looks like these paths are supported through gNMI.